tests/integration-tests/tests/schedulers/test_slurm.py

```diff
@@ -393,33 +411,57 @@ def _test_status_check_replacement(
     static_nodes, dynamic_nodes = assert_initial_conditions(
         scheduler_commands, num_static_nodes, num_dynamic_nodes, partition, job_id
     )
-    # Get network interface name from Head node, assuming Head node and Compute are of the same instance type
-    interface_name = remote_command_executor.run_remote_command(
-        "nmcli device status | grep ether | awk '{print $1}'"
-    ).stdout
-    logging.info("Detaching network interface {} on Compute nodes".format(interface_name))
-    # Only use dynamic nodes to test status check behavior
-    # Because static nodes will likely be placed in DOWN before status check failure shows in EC2
-    # Submit job that will detach network interface on all dynamic nodes, this will cause EC2 status check to fail
-    kill_job_id = scheduler_commands.submit_command_and_assert_job_accepted(
-        submit_command_args={
-            "command": "sudo ifconfig {} down && sleep 600".format(interface_name),
-            "partition": partition,
-            "constraint": "dynamic",
-            "other_options": "-a 1-{} --exclusive --no-requeue".format(num_dynamic_nodes),
-        }
+    # Assert that the default SlurmdTimeout=180 is in effect
+    _assert_slurmd_timeout(remote_command_executor, timeout=180)
+    # Nodes with networking failures should fail slurm health check before failing ec2_status_check
+    # Test on freshly launched dynamic nodes
+    kill_job_id = _submit_kill_networking_job(
+        remote_command_executor, scheduler_commands, partition, node_type="dynamic", num_nodes=num_dynamic_nodes
+    )
+    time.sleep(15)
+    scheduler_commands.cancel_job(kill_job_id)
+    # Assert dynamic nodes are in DOWN
+    _wait_for_compute_nodes_states(scheduler_commands, dynamic_nodes, expected_states=["down*", "down"])
+    # Assert dynamic nodes are reset
+    _wait_for_node_reset(scheduler_commands, static_nodes=[], dynamic_nodes=dynamic_nodes)
+    assert_num_instances_in_cluster(cluster_name, region, len(static_nodes))
+    # Assert ec2_status_check code path is not triggered
+    assert_no_msg_in_logs(
```

Comment:
this is a weak assertion. If we change the log message this becomes useless and we won't even realize. Is there any more robust check we can perform? for example assert for the presence of an error message? if not we could leave it as is

Reply:
Great point, adding assertion to assert messages in slurmctld log to check that slurm took action to mark the node as DOWN